cmd/consts/dymension.go

```diff
@@ -7,7 +7,7 @@ var MainnetHubData = HubData{
 	RpcUrl:        "https://dymension-mainnet-tendermint.public.blastapi.io",
 	ArchiveRpcUrl: "https://dymension-mainnet-tendermint.public.blastapi.io",
 	GasPrice:      "7000000000",
-	DaNetwork:     CelestiaMainnet,
+	DaNetwork:     AvailMainnet,
```

Comment:
please remove occurances where avail is set as the default da to replace celestia, currently it's not tested/integrated into roller and isn't considered stable